src/test/java/com/restassured/exercise/Skills.java

```diff
@@ -0,0 +1,104 @@
+package com.restassured.exercise;
+
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
+import static org.hamcrest.Matchers.is;
+import static org.testng.Assert.assertEquals;
+
+import java.util.List;
+
+import io.cucumber.java.en.Given;
+import  io.restassured.*;
+
+import org.json.simple.JSONObject;
+import org.junit.runner.Request;
+import org.testng.annotations.Test;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import io.restassured.specification.RequestSpecification;
+
+
+public class Skills {
+
+	//private JsonPath jsonPathEvaluator;
+
+	@Test
+
+	public void listSkills() {
+
+		RestAssured.baseURI = "https://parseapi.back4app.com";
```

Comment:
The base URL is replicated multiple times in different files, which may be a problem in the future if this URL changed.
Please move this string to a different place, so that if in the future it needed to be modified, the change would need to be done in a single file.
This can be done in different ways.
1. You can create a different class to store this kind of values and reference if from the Test Classes.
2. You can create an abstract class with a `protected` field referencing the baseURI `RestAssured.baseURI`, and make the Test classes inherit from that one, in this way all the "child" classes have access to the protected field.

Reply:
Changes made locally and will be loaded in the next PR.